### DISCUSSION TITLE
how to show day in month with leading zero?

### DISCUSSION THREAD

**Discussion by ehula** [fastfetch-cli/fastfetch#1305] [Q&A]
It is possible to display month, hour, minute and second with a leading zero (pretty), but it does not seem possible to display day with a leading zero. It seems like `day in month with leading zero - day-pretty` is missing.

Am I just not seeing how to do this? Is there a way to achieve this that I am missing? If not, feature request please.

Thanks.

**Comment by CarterLi** [comment]
Fixed in the dev build
